drivers/sdl/SCsub

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+
+Import("env")
+
+if "sdl" in env and env["sdl"]:
```

Comment:
My read on sccon environments and a quick check of its usage in the project says that the falsy check is unneeded, so this could just be 

But I'd like someone who actually uses sccon more than me to verify this

```suggestion
if "sdl" in env:
```